templates/etc/admin/admin.conf.erb

```diff
@@ -0,0 +1,87 @@
+# The pulp server configuration
```

Comment:
I'd add an indicator that this file is managed by puppet at the top similar to -- https://github.com/theforeman/puppet-foreman/blob/master/templates/_header.erb